### ISSUE TITLE
LoRA merge abnormal size increase.

### ISSUE THREAD

**Issue by Akegarasu** [hako-mikan/sd-webui-supermerger#19]
When I was merging the lora, I noticed an abnormal increase in the size of the model.
For example，finalprune 4G became 5G (final-prune+yuka-000005_old.ckpt).

I found it's because when merging lora to sd model, lora was loaded in float64. This results in all weights which lora has been applied will be cast to float64.
[pluslora.py#L328](https://github.com/hako-mikan/sd-webui-supermerger/blob/main/scripts/mergers/pluslora.py#L328)

I don't know if there is a deeper meaning here, but the default for sd-scripts should be float32.

Changed to fp32 and fixed this issue (final-prune+yuka-000005.ckpt) .
![image](https://user-images.githubusercontent.com/36563862/222175566-1c2361a8-ec3e-4e78-9ea9-63129b039e6e.png)

**Comment by hako-mikan** [comment]
Thanks for the report.

This is a very difficult problem and I spent several days thinking about it. 
In making this script, I tried to reproduce pastel-mix(LoRA merged model) in verifying this feature, but could not reproduce it. 
Finally, I found that setting a value 0.01 off the LoRA ratio set in pastel-mix, I succeeded reproduce pastel mix. 
I thought this was an error problem and tried to solve it by setting 64. 
Although it can not be reproduced, it was an error of the degree that xformers produced (of course, xformers was turned off at the time of verification)
It seemed to work at the time of verification, but when I tried it now, it still seemed to create an difference. 
So I took your advice and set it to a normal float. Thank you very much.
